homeassistant/components/mqtt/strings.json

```diff
@@ -342,9 +376,77 @@
     }
   },
   "selector": {
+    "device_class_sensor": {
```

Comment:
OT for this change. But this is going to get messy fast if more config flow integrations need to select a device class (and/or units)

Reply:
True, already discussed this with Erik and Joost. But not sure we have an alternative yet.